mrp_component_operation/views/mrp_component_operation_views.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_mrp_component_operation_form" model="ir.ui.view">
+        <field name="name">view_mrp_component_operation_form</field>
+        <field name="model">mrp.component.operation</field>
+        <field name="arch" type="xml">
+            <form string="Component Operation">
+                <sheet>
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        invisible="active"
+                    />
+                    <h1>
```

Comment:
The text has been shortened

v17:
![image](https://github.com/user-attachments/assets/e5d8389a-d845-4594-b795-ee3844e17396)

v14:
![image](https://github.com/user-attachments/assets/003be308-efe1-460c-a71a-31350df997b8)

Reply:
Added `<div class="oe_title">` to ensure the same behavior